scipy/stats/_continuous_distns.py

```diff
@@ -5930,29 +5930,26 @@ def _argcheck(self, df1, df2, nc):
     def _rvs(self, dfn, dfd, nc, size=None, random_state=None):
         return random_state.noncentral_f(dfn, dfd, nc, size)
 
-    def _pdf_skip(self, x, dfn, dfd, nc):
+    def _pdf(self, x, dfn, dfd, nc):
```

Comment:
Find the distribution in `_continuous_distns.py` (or `_discrete_distns.py` for discrete distributions) and use the `_boost` ufuncs to implement all underscored `rv_continuous` (or `rv_discrete`) methods